dc/s2n-quic-dc/src/path/secret/map/test.rs

```diff
@@ -325,10 +325,16 @@ fn no_memory_growth() {
 }
 
 #[test]
-#[cfg(all(target_pointer_width = "64", target_os = "linux"))]
 fn entry_size() {
+    let mut should_check = true;
+
+    should_check &= cfg!(target_pointer_width = "64");
+    should_check &= cfg!(target_os = "linux");
+    should_check &= std::env::var("S2N_QUIC_RUN_VERSION_SPECIFIC_TESTS").is_ok();
+    should_check &= std::env::var("S2N_QUIC_PLATFORM_FEATURES_OVERRIDE").is_err();
+
     // This gates to running only on specific GHA to reduce false positives.
-    if std::env::var("S2N_QUIC_RUN_VERSION_SPECIFIC_TESTS").is_ok() {
-        assert_eq!(fake_entry(0).size(), 238);
+    if should_check {
+        assert_eq!(fake_entry(0).size(), 270);
```

Comment:
Hm, this seems a bit unfortunate. Do we know what is driving the increase here? I thought I checked that we were running the test before -- maybe it switched to a different CI worker?

Reply:
oh i actually think this is a merge error... lemme look more into it

Reply:
yeah i messed up the merge there. pushing a new one.